src/eko/io/runcards.py

```diff
@@ -48,10 +46,16 @@ class TheoryCard(DictLike):
     xif: float
     """Ratio between factorization scale and process scale."""
     n3lo_ad_variation: N3LOAdVariation
-    """|N3LO| anomalous dimension variation: ``(gg_var, gq_var, qg_var, qq_var)``."""
+    """|N3LO| anomalous dimension variation: ``(gg_var, gq_var, qg_var,
+    qq_var)``."""
```

Comment:
```suggestion
    """|N3LO| anomalous dimension variation: ``(gg, gq, qg,qq)``."""
```

I seem to remember someone (pylint or black?) was complaining if the short description was not in one line (but maybe I misremember ...)

Reply:
No one of them, but `pydocstyle`.

This is the formatter that it is instead adding the newline, because the line was too long...